src/App.tsx

```diff
@@ -1,24 +1,14 @@
 import React, { FC } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { CssBaseline } from '@material-ui/core';
+import { ApolloProvider } from '@apollo/client';
+import apiGraphql from './Services/GraphClient';
```

Comment:
I suggest moving to `/helpers`.

```suggestion
import apolloClient from './helpers/apolloClient';
```

Reply:
Resolved on 583001d167d9742054bd0cf9dd64e843bb5b8a7d